pwndbg/lib/kernel/kconfig.py

```diff
@@ -99,7 +99,7 @@ def __getattr__(self, name: str):
     @property
     def CONFIG_SLUB_TINY(self) -> bool:
         krelease = pwndbg.aglib.kernel.krelease()
-        if krelease is not None and krelease < (6, 2): # config added after v6.2
+        if krelease is not None and krelease < (6, 2):  # config added after v6.2
```

Comment:
I think you may have accidentally formatted files beyond what was touched in your PR?

Reply:
Weird, shouldn't this use our/hardcoded versions of linters etc?

(Also we don't update uv here)

Shall those changes maybe be done in another PR? I'm okay merging them here tho